doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -703,6 +706,11 @@ Bluetooth libraries and services
 
   * Added the :c:func:`bt_scan_update_connect_if_match` function to update the autoconnect flag after a filter match.
 
+* :ref:`bt_le_adv_prov_readme`:
+
+  * By default, the Bluetooth device name provider provides data only in the pairing mode (:c:member:`bt_le_adv_prov_adv_state.pairing_mode`).
+    The newly introduced Kconfig option (:kconfig:option:`CONFIG_BT_ADV_PROV_DEVICE_NAME_PAIRING_MODE_ONLY`) can be disabled to provide the device name also when the device is not in the pairing mode.
+
```

Comment:
Sorry, my bad, I completely misread the heading. As @peknis suggested, we should align this with the entire section

```suggestion
* :ref:`bt_le_adv_prov_readme`:

  * Added an option to provide the device name also when the device is not in the pairing mode by disabling the newly introduced Kconfig option (:kconfig:option:`CONFIG_BT_ADV_PROV_DEVICE_NAME_PAIRING_MODE_ONLY`).
  By default, the Bluetooth device name provider provides data only in the pairing mode
```
tbc if the meaning is correct

Reply:
To be honest, I would prefer to keep the information about modified default behavior as the first one. The Kconfig option was introduced to allow easily brining back the previous behavior if needed.